chef/cookbooks/cpe_autopkg/resources/cpe_autopkg_update.rb

```diff
@@ -25,6 +25,8 @@
 
 action :update do
   return unless node['cpe_autopkg']['update']
+  return unless ::File.exist?(AUTOPKG_BIN)
```

Comment:
You may want to log, troubling why this is exiting would be really hard otherwise.